File: DailyFlash Solutions/DailyFlash Solutions/00-DailyFlash_Solutions/13_Jan_Solution_Two/Java/prog5.java
import java.io.*;
class Demo{
	public static void main(String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		System.out.println("Input : ");
		int n1 = Integer.parseInt(br.readLine());
		System.out.println(n1%5==0 && n1%11==0?"Output : Divisible":"Output : Not divisible");
	}
}

File: DailyFlash Solutions/DailyFlash Solutions/00-DailyFlash_Solutions/13_Jan_Solution_Two/C++/prog5.cpp
#include<iostream>

int main() {
	int n1;
	std::cout<<"Input : ";
	std::cin>>n1;
	n1%5==0 && n1%11==0?std::cout<<"Output : Divisible":std::cout<<"Output : Not divisible";
	return 0;
}
